projects/storefrontapp-e2e-cypress/cypress/helpers/navigation.ts

```diff
@@ -5,15 +5,20 @@ export const navigation = {
   }: {
     queryStringParams?: string;
     options?: Partial<Cypress.VisitOptions>;
-  }): void {
+  }): Cypress.Chainable<Window> {
     queryStringParams =
       !queryStringParams || queryStringParams.indexOf('?') !== -1
         ? queryStringParams
         : `?${queryStringParams}`;
-    cy.visit(`/${queryStringParams ? queryStringParams : ''}`, options);
+    return cy.visit(`/${queryStringParams ? queryStringParams : ''}`, options);
   },
-  goToProduct(id): void {
-    cy.visit(`/product/${id}`);
+  goToProduct(id): Cypress.Chainable<Window> {
+    return cy.visit(`/product/${id}`);
+  },
+  waitForPage(page: string): Cypress.Chainable<Cypress.WaitXHR> {
```

Comment:
what's the reason for putting this here and not re-using the one from the other helper function

I don't mind as well though!

Reply:
@bgambocjaviniar I can use the other one, but there were 2 reasons for sticking it here:
1: I needed to return the cy command to chain (I no longer use this, however)
2: It felt strange having a dependency to the checkout-flow, even though I am not testing anything with the checkout

I can change this back if you would like

Reply:
I guess it's fine to keep it, but we're going to have a refactor in the future to remove dependencies as you mentioned and create a proper e2e architecture